src/sentry/event_manager.py

```diff
@@ -2455,6 +2459,68 @@ def _save_aggregate_performance(jobs: Sequence[PerformanceJob], projects: Projec
                 EventPerformanceProblem(event, performance_problems_by_hash[problem_hash]).save()
 
 
+@metrics.wraps("save_event.send_occurrence_to_platform")
+def _send_occurrence_to_platform(jobs: Sequence[Job], projects: ProjectsMapping) -> None:
+    for job in jobs:
+        event = job["event"]
+        project = event.project
+        event_id = event.event_id
+
+        use_platform = options.get("performance.issues.send_to_issues_platform", False)
+        if use_platform:
+            per_project_option = project.get_option(
+                "sentry:performance_issue_send_to_issues_platform", False
+            )
+            if per_project_option:
+                performance_problems = job["performance_problems"]
+                for problem in performance_problems:
+                    # handle only N+1 db query detector first
+                    if problem.type.type_id == PerformanceNPlusOneGroupType.type_id:
+                        occurrence = IssueOccurrence(
+                            id=uuid.uuid4().hex,
+                            resource_id=None,
+                            project_id=project.id,
+                            event_id=event_id,
+                            fingerprint=[problem.fingerprint],
+                            type=problem.type,
+                            issue_title=problem.title,
+                            subtitle=event.transaction,
+                            # TODO use this to set metadata.value="problem.desc"
+                            evidence_data={"value": problem.desc},
+                            evidence_display=[
+                                IssueEvidence(
+                                    name="parent_span_ids",
+                                    value=problem.parent_span_ids,
+                                    important=True,
+                                ),
+                                IssueEvidence(
+                                    name="cause_span_ids",
+                                    value=problem.cause_span_ids,
+                                    important=False,
+                                ),
+                                IssueEvidence(
+                                    name="offender_span_ids",
+                                    value=problem.offender_span_ids,
+                                    important=False,
+                                ),
```

Comment:
Do we want the span ids to be displayed by default, or are they just used by the UI to render things?

If we just use them as data, then it might make sense to include these in `evidence_data` instead.

![Screen Shot 2023-03-14 at 9 19 51 AM](https://user-images.githubusercontent.com/6288560/224845645-293532a5-b262-4c38-9e52-481ca89bea24.png)
![Screen Shot 2023-03-14 at 9 23 50 AM](https://user-images.githubusercontent.com/6288560/224846196-e9ebb5bd-4dfc-4671-936c-0a3efd7a5666.png)

Based on the UI and email^, I think we want to have the actual span details in the `evidence_display`, and then the span ids in `evidence_data`

Reply:
Although I'm not sure how generalizable that is to all perf issue types. But ideally, `evidence_display` has things we can display to the user, so that we can include them in email/slack/etc without having to customize for every new issue type.

Reply:
that makes sense. I update both evidence sections. I'm passing span ids because we look their descriptions on the client side but maybe that's not ideal

Reply:
I'm wondering here if we should automatically translate the span ids to the span contents (just for `evidence_display`). That way it'll automatically display correctly. Then anything the FE client code needs to do with span ids can just work from `evidence_data`. I'm not totally sure if that makes sense?

This could be a thing where maybe we add `parent_spans, cause_spans, offender_spans` to the `PerformanceProblem`? And then we extract those from the transaction in the detector.
Since in the future, we won't actually be storing `PerformanceProblem` in nodestore at all, it'll just be used as an interface for you to send things to the issue platform.

Or maybe we should just emit `evidence_data` and `evidence_display` explicitly from the detectors. Then we'd be able to just shove it straight into evidence and it'd just work. I'm not sure if that makes sense here, but it means that each performance detector could more easily customize what they want to display.

Reply:
yeah I think `evidence_data` and `evidence_display` should be explicitly emitted from the detectors. So that I don't have to translate PerformanceProblem output into evidence because it'd be different for each detector. I'm going to add that change in 👍

Reply:
going back to this, we might need to go back to passing span ids in evidence display because we do some extra rendering customization like turning transaction name into a link and highlighting parameters in the url for http span description. adding some markup like that here is pretty cumbersome so I wonder if we need to go back to passing span ids and letting the client render the evidence sections as it needs
<img width="925" alt="Screenshot 2023-03-15 at 1 56 18 PM" src="https://user-images.githubusercontent.com/23648387/225401125-2a863236-c7e5-49fe-892d-f58c7f26fb2c.png">
<img width="1054" alt="Screenshot 2023-03-15 at 1 56 52 PM" src="https://user-images.githubusercontent.com/23648387/225401127-717eba1c-61b3-48eb-9e8a-807f2afe2972.png">

Reply:
That's all basically UI customization, so in that case you're better off using the span ids from `evidence_data` and ignoring the `evidence_display` entirely. Then `evidence_display` can just be used for emails/slack (unless you also want to customize there)

Reply:
okay, then I'll keep everything as is 👍